test/tests/functional/pbs_snapshot_unittest.py

```diff
@@ -94,12 +95,22 @@ def setup_sc(self, sched_id, partition, port):
         :param log_filter: log filter value for the scheduler
         :type log_filter: int
         """
-        a = {'partition': partition,
-             'sched_host': self.server.hostname,
-             'sched_port': port}
+        if sched_priv is not None:
+            a = {'partition': partition,
+                 'sched_host': self.server.hostname,
+                 'sched_port': port,
+                 'sched_priv': os.path.join(sched_priv, 'sched_priv_sc3'),
```

Comment:
Please don't hard-code the priv and log directory names inside the generic function setup_sc(), pass it the complete path from the caller itself.